plugins/src/lib.rs

```diff
@@ -526,17 +526,17 @@ where
             // the user-code, which may require some cleanups or
             // similar.
             tokio::select! {
-                e = self.dispatch_one(&mut input, &self.plugin) => {
-                    if let Err(e) = e {
-			return Err(e)
-                    }
-		},
-		v = receiver.recv() => {
-                    output.lock().await.send(
-			v.context("internal communication error")?
-                    ).await?;
-		},
-            }
+                    e = self.dispatch_one(&mut input, &self.plugin) => {
+                        if let Err(e) = e {
+                return Err(e)
+                        }
+            },
+            v = receiver.recv() => {
+                        output.lock().await.send(
+                v.context("internal communication error")?
+                        ).await?;
+            },
+                }
```

Comment:
These indentations look terribly broken.
Is this by intention? (I'm a RUST noob)

Reply:
Yeah, it's rather messy, since it is internally actually expanding a macro, which is then formatted apparently. This also doesn't seem to be stable, in the sense that two different crate versions or `cargo` versions will format this differently. I'm looking into how we could remove that `select!` statement which is causing this mess, replacing it with a single task or a chained task with a timeout or something.